code-challenges.test.js

```diff
@@ -24,25 +29,90 @@
 
 // b) Create the function that makes the test pass.
 
+//create fib variable
+//for fib define num values 
+//for loop states that the function will act on numbers greater than 2, but less than or equal to the example inputs (6,10). Function will iterate through the given parameters
+//return array of 6 and 10 in fib.
+
+
+
+
+var i;
+var fib = [];
+
```

Comment:
I see some good code is here. When looking at the question it shows, that index 0 and index 1 will both have the value of 1. 
// Example input: 6
// Expected output: [1, 1, 2, 3, 5, 8]

// Example input: 10
// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]